homeassistant/components/tesla_wall_connector/sensor.py

```diff
@@ -120,10 +120,10 @@ class WallConnectorSensorDescription(
         entity_category=ENTITY_CATEGORY_DIAGNOSTIC,
     ),
     WallConnectorSensorDescription(
-        key="total_energy_kWh",
-        name=prefix_entity_name("Total Energy"),
-        native_unit_of_measurement=ENERGY_KILO_WATT_HOUR,
-        value_fn=lambda data: data[WALLCONNECTOR_DATA_LIFETIME].energy_wh / 1000.0,
+        key="energy_kWh",
```

Comment:
I can see that `kilo` prefix is still in there

Reply:
Will fix in next update :)